Swift/DatabaseConfiguration.swift

```diff
@@ -57,4 +57,10 @@ public struct DatabaseConfiguration {
         return config
     }
     
+    init(_ impl: CBLDatabaseConfiguration) {
```

Comment:
Same as above, if database is passed, when creating a collection, I think this is not needed.